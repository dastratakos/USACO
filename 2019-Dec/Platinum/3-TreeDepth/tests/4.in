8 10 841786427
