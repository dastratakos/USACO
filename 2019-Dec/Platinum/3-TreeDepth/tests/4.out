5233 6569 7487 8150 8608 8851 8810 8275
